packages/yoshi-common/src/webpack.config.ts

```diff
@@ -670,6 +714,14 @@ export function createBaseWebpackConfig({
             ]
           : []
         : []),
+
+      ...(isProduction && !inTeamCity
+        ? [
+            new WebpackBar({
+              ...getProgressBarInfo(configName, isDev, isMonorepo, name),
+            }),
```

Comment:
Should work the same, no?

```suggestion
            new WebpackBar(
              getProgressBarInfo(configName, isDev, isMonorepo, name),
            ),
```